junebug/stores.py

```diff
@@ -19,32 +20,44 @@ def __init__(self, redis, ttl=None):
 
     @inlineCallbacks
     def _redis_op(self, func, id, *args, **kwargs):
+        ttl = kwargs.pop('ttl')
+        if ttl is None:
+            ttl = self.ttl
```

Comment:
I think to make this work we'd need to use an explicit sentinel value, e.g.

```
USE_DEFAULT_TTL = object()

def _redis_op(...):
    ttl = kwargs.pop('ttl')
    if ttl is self.USE_DEFAULT_TTL:
        ttl = self.ttl

def get_id(..., ttl=USE_DEFAULT_TTL):
    ...
```

Too complicated?